lib/utils.h

```diff
@@ -48,9 +49,8 @@ inline static bool isPrefixStringCharLiteral(const std::string &str, char q, con
 
 inline static bool isStringCharLiteral(const std::string &str, char q)
 {
-    for (const std::string & p: {
-    "", "u8", "u", "U", "L"
-}) {
+	static const std::vector<std::string> suffixes{"", "u8", "u", "U", "L"};
```

Comment:
This will not be threadsafe as far as I see. A static object in a function.

Reply:
Ah right. Sorry about that.

Reply:
Apparently this is no longer an issue in C++11 - see https://stackoverflow.com/questions/335369/finding-c-static-initialization-order-problems/335746#335746

I found it curious since I have been using variables in this way for a while now.

Reply:
interesting .. it's great news of course but I assume there will be a cost.

Reply:
Probably neglect ale with small code bases that do not call this often. If they don't call it at all there's no cost and for big ones there's a win.